### DISCUSSION TITLE
Detection Results

### DISCUSSION THREAD

**Discussion by dolfMo** [ultralytics/ultralytics#9497] [Q&A]
Hello😊! I use the YOLOv8 object detection model to obtain detection results.
1.When the target in the image is too large, the detected label results will not be displayed completely or even not, only the annotation box. I want to know how to solve this problem.

**Comment by glenn-jocher** [comment]
Hello! 😊 Thanks for using YOLOv8 for your object detection tasks! If you're encountering issues with large objects where the detection labels aren't fully visible, it might help to adjust the label display settings in your visualization code. One approach is to ensure the label is drawn inside or on top of the bounding box, depending on the object's size. Here's a snippet for adjusting label visibility in your results plotting:

```python
from ultralytics import YOLO

# Load your model
model = YOLO('yolov8n.pt')

# Perform detection
results = model('path/to/your/image.jpg')

# Customize visualization
results.render()  # or results.show(), results.save() etc.
```

For finer control, you may need to modify the `plot()` method within the YOLOv8 codebase to dynamically adjust text position based on object sizes and frame boundaries. If working with the CLI or exploring the source for direct adjustments isn't ideal, reviewing the documentation or code comments around result visualization could offer additional insights.

Happy detecting, and let us know how it goes!
